build.gradle

```
@@ -24,6 +24,9 @@ buildscript {
         // Required for gradle liquibase plugin
         classpath ('org.liquibase:liquibase-core:4.29.1')
     }
+    ext {
+        swaggerUIVersion = "5.17.14";
```

We're not consistent about this, but I think the convention is to only use `"` for interpolated strings
```suggestion
        swaggerUIVersion = '5.17.14';
```